File: java/8.7/example2.java
import java.util.Random;

public class example2 {
    public static void main(String[] args) {
        Random rand = new Random();
        Integer[] arrayData = new Integer[101];
        for (Integer y = 0; y <= 100; y++) {
            arrayData[y] = rand.nextInt(25565);
        } 
        Integer total = 0;
        for (Integer count = 0; count < 100; count++) {
            total = total + arrayData[count];
        }
        System.out.println(total);
    }
}

File: python/8.7/example2.py
from random import randint # Importing ranint from random 

arrayData = [] # Setting the value of array data to []
total = 0 # Setting the value of total to 0

for x in range(1, 101): # Running a loop 100 times
    arrayData += [randint(0, 25565)] # Adding a random number to the array of arrayData

for count in range(0, 100): # Running a loop 100 times
    total = total + arrayData[count] # Adding the number in the arrayData taht corisponds with count to the total

print(str(total)) # Printing the string of total